test/e2e/performanceprofile/functests/utils/label/label.go

```diff
@@ -0,0 +1,38 @@
+package label
+
+// The label package contains a list of labels that can be used in
+// the functests to indicate and point certain behavior or characteristics
+// of the various tests.
+// Those can be filtered/focused by ginkgo before the test runs.
+
+const (
+	// Slow means test that usually requires reboot or takes a long time to finish.
+	Slow = "slow"
+
+	// ReleaseCritical is for tests that are critical for a successful release of the component that is being tested.
+	ReleaseCritical = "release-critical"
+
+	// SpecializedHardware is for tests that need special Hardware like SR-IOV devices, specific NICs, etc.
+	SpecializedHardware = "specialized-hardware"
+)
+
+// FeatureTag is a label indicates the feature that being tested.
+type FeatureTag string
```

Comment:
why a type for feature tags and not for the consts (e.g. `Slow`) above?

Reply:
I couldn't find a good name to classify the set about :)

Reply:
maybe just `Label` ? it's a overused name though.

Reply:
May be we can classify them as below

**Tiered** Tests  Tests covering different levels (tier0,tier1) 
**Featured **  Tests for specific features  (memorymanager, mixedcpus, workloadhints)
**Release Critical** tests Tests which are critical for tests 
**HardwareCompatibility**  Tests for testing hardware compatibility (sriov, GPU, hugepages) ?

All the tests should be under some label by default

Reply:
@mrniranjan 
Do we have a concrete division for the different tiers?

Reply:
Yes we do have:
**tier0:** 
Definition :
Automated unit tests
Minimal time needed to execute (minutes to 1 hour)
Process criteria :
100% automated, must pass 100%
Development maintains tests and reviews results

**tier1** (component)
Definition :
Component level functional tests
Minimal time needed to execute (minutes to hours)
Process criteria:
Executed after tier 0 passing
100% automated and must pass 100%
QE / Development maintains tests and reviews results

**tier2** (integration)
Definition:
Integration level functional tests
May include basic non-functional tests (security, performance regression, install, compose validation)
Runs during nightly time frame
Process criteria :
Executed after components pass tier 1 testing
100% automated and must pass 100%
QE  maintains tests and reviews result

**tier3**  (system)
Definition:
System, scenario and  non-functional tests (which includes Fault Tolerance / Recovery / Failover)
Test that don't fit in tier 2 due to time, complexity, and other  factors
Process criteria :
Executed after components pass tier 2 testing
System or scenario testing doesn't block Non-functional tests. They can be executed in parallel.
100% automated
QE maintains tests and reviews results

Reply:
@mrniranjan  @Tal-or  can we add this information at least in one of our u/s docs ?  maybe even create a testing u/s doc under https://github.com/openshift/cluster-node-tuning-operator/tree/master/docs/performanceprofile ?

Reply:
yes , the above is more standard definitions, but we can make our own definitions of tiered tests and document them.

Reply:
@mrniranjan 
Thank you for the valuable information, i'll add the tiers then.

As @ffromani the advantage of having specific types and not plain `strings` is that we can force specific labels and not random strings.


>All the tests should be under some label by default

We need to develop a way to force that. probably some GH action